packages/types/api/rules.d.ts

```diff
@@ -36,6 +36,9 @@ export interface NewRuleRecord extends PartialRuleRecord {
 }
 
 export interface RuleRecord extends NewRuleRecord {
+  executionNamePrefix?: string,
+  queueUrl?: string,
+  tags?: string[],
```

Comment:
I think these properties belong on the `PartialRuleRecord`, because that type describes a rule record before it is saved to the database, which may include these properties. This type describes a rule record should look like when retrieved from the database, at which popint `createdAt` and `updatedAt` have been set and thus are required

Reply:
Noted